### ISSUE TITLE
Crash: MacOS 3.00+ (d1028510c1)

### ISSUE THREAD

**Issue by GaryElshaw** [transmission/transmission#3155]
Usually within the first minute of starting.

[crash-3.00+d1028510c1.txt](https://github.com/transmission/transmission/files/8792886/crash-3.00%2Bd1028510c1.txt)

**Comment by robd003** [comment]
Seeing the exact same thing. Looks like https://github.com/transmission/transmission/commit/d1028510c1c49ceef1dc2445a0c0115b892bc6ad broke it

**Comment by GaryElshaw** [comment]
Yep :-(

**Comment by sergserg2** [comment]
Same on ARM64, with many new warnings

```
[2022-05-29 13:18:21.723] WRN torrent-metainfo.cc:514 unexpected: path '[][info][files][][filehash]', str '▒▒▒A>Ǖ]d٨s▒▒▒%' (/home/software/transmission/transmission-3.00+-rd1028510c1/libtransmission/torrent-metainfo.cc:514)
[2022-05-29 13:18:21.723] WRN torrent-metainfo.cc:514 unexpected: path '[][info][files][][filehash]', str '▒)$▒ %:▒▒f:Z▒~pG' (/home/software/transmission/transmission-3.00+-rd1028510c1/libtransmission/torrent-metainfo.cc:514)
[2022-05-29 13:18:21.723] WRN torrent-metainfo.cc:514 unexpected: path '[][info][files][][filehash]', str '▒͢1▒▒G▒▒Aڌ▒5X▒q&' (/home/software/transmission/transmission-3.00+-rd1028510c1/libtransmission/torrent-metainfo.cc:514)
[2022-05-29 13:18:21.723] WRN torrent-metainfo.cc:514 unexpected: path '[][info][files][][filehash]', str '▒K;▒{
   -▒3▒S▒VF▒▒p' (/home/software/transmission/transmission-3.00+-rd1028510c1/libtransmission/torrent-metainfo.cc:514)
```

**Comment by ckerr** [comment]
Duplicate of #3153

**Comment by ckerr** [comment]
@sergserg2 please file a new issue for these warnings. They're separate from the crash and probably ought to be handled in a separate PR.

tl;dr of what's going on tthere is in the description of https://github.com/transmission/transmission/pull/3133
